### ISSUE TITLE
Control over the spinner when using concurrency

### ISSUE THREAD

**Issue by mwgamble** [xolox/python-executor#2]
It would be nice to have some level of control over the spinner that is displayed when using concurrency. For example, I would like to be able to have it re-render the spinner only once every 1-2 seconds, rather than constantly.

**Comment by mwgamble** [comment]
Just wanted to acknowledge your ongoing commitment to the quality of your software. I truly appreciate you caring about such an old issue and look forward to the point where executor is updated accordingly to match the humanfriendly updates <3

**Comment by xolox** [comment]
Hi Matthew, first of all thanks for the feedback. Second, sorry it took me so long to get back to this! I've just released [executor 23.1](https://executor.readthedocs.io/en/latest/changelog.html#release-23-1-2020-05-14) which enables fine grained control over the spinner shown by command pools. Here's a silly example I used to confirm that it works as intended, this assumes Python 3:

```python
from executor import ExternalCommand
from executor.concurrent import CommandPool
from humanfriendly.terminal.spinners import Spinner

pool = CommandPool(spinner=Spinner(glyphs=['◐', '◓', '◑', '◒'], interval=0.5))
pool.add(ExternalCommand('sleep', '20'))
pool.add(ExternalCommand('sleep', '15'))
pool.add(ExternalCommand('sleep', '10'))
pool.run()
```

I'm going to close this issue now because I think I've adequately implemented your feature request, but if you disagree feel free to reopen this issue or open a new one. Cheers!

**Comment by mwgamble** [comment]
No need to apologise :) Thank you so much!
